drivers/dma/Kconfig.intel_adsp_gpdma

```diff
@@ -7,6 +7,7 @@ config DMA_INTEL_ADSP_GPDMA
 	bool "Intel ADSP General Purpose Direct Memory Access driver"
 	default y
 	depends on DT_HAS_INTEL_ADSP_GPDMA_ENABLED
+	depends on PM_DEVICE_RUNTIME
```

Comment:
I don't think the driver should be dependent on runtime pm. It should be optional, if PM is not enabled the device is always available.